docs/rfcs/0001-video-retranscoding.md

```diff
@@ -0,0 +1,43 @@
+## Video Re-transcoding
+
+#### Abstract
+Provide a way to trigger re-transcoding of specified videos, or all videos in a collection, if new presets are needed.
+
+
+#### Architecture Changes
+
+- Create a flag on Collection and Video: boolean field ‘schedule_retranscode’
+- Create a recurring task to look for videos and collections with `schedule_retranscode=True`.
+  - check flagged collections first:
+     - set `schedule_retranscode=True` for every Video in the Collection, then
+     - set `schedule_retranscode=False` on the Collection
+  - Trigger a new encode job for each flagged Video.
+  - Assign a new video status - ‘Retranscoding’
```

Comment:
Is "video status" an existing field on the video model?

Reply:
Yes, this is just adding a new possible value for it.

Reply:
adding a new failure status too - 'Retranscode failed'